Sources/Deprecations+Removals.swift

```diff
@@ -2,200 +2,10 @@ import Foundation
 import Dispatch
 import Result
 
-// MARK: Unavailable methods in ReactiveSwift 2.0.
-extension AnyDisposable {
-	@available(*, unavailable, renamed:"init(_:)")
-	public convenience init(action: @escaping () -> Void) { fatalError() }
-}
-
+// MARK: Unavailable methods in ReactiveSwift 3.0.
 extension Signal {
-	@available(*, unavailable, renamed:"promoteError")
-	public func promoteErrors<F>(_: F.Type) -> Signal<Value, F> { fatalError() }
-
 	@available(*, unavailable, message:"Use the `Signal.init` that accepts a two-argument generator.")
 	public convenience init(_ generator: (Observer) -> Disposable?) { fatalError() }
 }
 
-extension SignalProducer {
-	@available(*, unavailable, renamed:"promoteError")
-	public func promoteErrors<F>(_: F.Type) -> SignalProducer<Value, F> { fatalError() }
-}
-
-extension Lifetime {
-	@available(*, unavailable, renamed:"hasEnded")
-	public var isDisposed: Bool { fatalError() }
-
-	@discardableResult
-	@available(*, unavailable, renamed:"observeEnded(_:)")
-	public func add(_ action: () -> Void) -> Disposable? { fatalError() }
-
-	@discardableResult
-	@available(*, unavailable, message:"Use `observeEnded(_:)` instead.")
-	public static func += (left: Lifetime, right: () -> Void) -> Disposable? { fatalError() }
-
-	@discardableResult
-	@available(*, deprecated, message:"Use `observeEnded(_:)` with a method reference to `dispose()` instead. This method is subject to removal in a future release.")
```

Comment:
Since this one was only `deprecated`, it should stay until the next major version and become `unavailable`.

Reply:
Oh overlooked that 🙈

Reply:
Addressed in 0a378cb